pyproject.toml

```diff
@@ -0,0 +1,106 @@
+[project]
```

Comment:
Should we also add `license` - it can point to file? 

https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license